.travis.yml

```diff
@@ -8,16 +8,13 @@ addons:
   google-chrome: latest
 
 before_script:
-  - gulp
-  - bundlesize -f dist/sweetalert2.all.min.js -s 15kB
+  - npm run build-dist
 
 script:
-  # unit tests against dist/sweetalert2.js
-  - xvfb-run testem ci --reporter dot
-
-  # unit tests against dist/sweetalert2.all.min.js
-  - sed -i 's/sweetalert2.js/sweetalert2.all.min.js/g' testem.json
-  - xvfb-run testem ci --reporter dot
+  - npm run check-bundlesize
+  - npm run check-lint
+  - xvfb-run npm run check-test
```

Comment:
With the `check-` prefix users can't use https://docs.npmjs.com/cli/test, do we really need it?

Reply:
Yes, this is the problem of my first TODO...

Travis will automatically run the "test" script before doing anything else (if there is one), and it does so *without* the `xvfb-run` command which is needed for the tests to work in the headless CI environment. This is why there is no "test" script at the moment.

My idea was to have the "test" script (as well as the "prepublish" script, which has the same problem currently) run *all* of the checks (bundle-size, linting, and actual tests) *as well as* run the build beforehand. In this way, someone can run just `npm t` to make sure all is good before pushing. 

Does my last paragraph sound good? Do you have any suggestion for alternate wording?

Reply:
> Travis will automatically run the "test" script before doing anything else (if there is one), and it does so without the xvfb-run command which is needed for the tests to work in the headless CI environment. This is why there is no "test" script at the moment.

`npm test` is the default build script which will not automatically happen in our case as we are defining our own and overriding the default one.